plutus-benchmark/script-contexts/test/9.6/checkScriptContext2.size.golden

```diff
@@ -1 +1 @@
-3119
\ No newline at end of file
+3157
```

Comment:
Any idea of the reason for the increased sizes?

Reply:
My guess is that it's that the new definitions are more inliner-friendly, which creates some inlining and monomorphization opportunities, which increases code size, see [this](https://github.com/IntersectMBO/plutus/pull/6530#discussion_r1783255465) comment.